planetary-social/nos
# #1583: Create a broken link view for images

## Issues covered
#1583 

## Description
Adds a broken link view and shows it for links that look like images, but fail to load. Note that this doesn't work for web pages or videos since those still use Apple's built-in link loader.

Note: The background color isn't quite right and I'm working on that now.

## How to test
1. Create a post with a broken link to what appears to be an image, or navigate to one. For example: `https://example.com/no-way.png`
2. Observe the broken link view
3. For three bonus points, try it in a gallery where some images are real actual working URLs and some aren't! (or just look at note169m6mwvf3texa65m3jczau300ljmp35gksl7ytpdlu5rach2sfyssrj8xm but you won't get any bonus points)

## Screenshots/Video

https://github.com/user-attachments/assets/bfebca45-2168-4d78-817d-1bae293442dc

https://github.com/user-attachments/assets/519ee0ba-0a0f-44bf-bfa5-6cf48604f081
